packages/activation-notice/src/app/components/test/step2.js

```diff
@@ -27,6 +27,11 @@ import { renderWithTheme } from '../../../testUtils';
 import Step2 from '../step2';
 import { ConfigProvider } from '../../config';
 
+jest.mock('@wordpress/element', () => ({
+  ...jest.requireActual('@wordpress/element'),
+  createInterpolateElement: (string) => string,
```

Comment:
To avoid repetition, I think we can just put this in `__mocks__/@wordpress/element.js`